File: Payhub - SDK/java_SDK/Samples/FindTransactionReportsSample.java
package Samples;

import java.io.IOException;
import java.util.List;

import com.payhub.ws.api.TransactionManager;
import com.payhub.ws.api.TransactionReportInformation;
import com.payhub.ws.api.TransactionType;
import com.payhub.ws.model.Merchant;
import com.payhub.ws.model.TransactionSearchParameters;

public class FindTransactionReportsSample {
	public void findTransactions() throws IOException{
		String url = "https://staging-api.payhub.com/api/v2/";
        String oauth = "107d74ab-4a18-4713-88ff-69bd05710086";
        Merchant merchant = new Merchant();
        merchant.organization_id = 10127;
        merchant.terminal_id = 215;	
        TransactionSearchParameters tsp = new TransactionSearchParameters();
        tsp.setAmountFrom("1");
        tsp.setAmountTo("1002");
		TransactionManager transaction = new TransactionManager(url, oauth, merchant);
		List<TransactionReportInformation> tri = transaction.findTransactions(tsp);

	}
}

File: Payhub - SDK/php_SDK/Samples/FindTransactionReportsSample.php
<?php
/**
 * Created by PhpStorm.
 * User: agustin
 * Date: 28/07/2015
 * Time: 16:51
 */

$path_to_IncludeClases="../com/payhub/ws/extra/includeClasses.php";
include_once $path_to_IncludeClases;
//Defining the Web Service URL
$WsURL="https://staging-api.payhub.com/api/v2/";
$oauth_token = "107d74ab-4a18-4713-88ff-69bd05710086";

//Defining data for the SALE transaction
// Merchant data (obtained from the payHub Virtual Terminal (3rd party integration)
$merchant = new Merchant();
$merchant->setOrganizationId(10127);
$merchant->setTerminalId(215);

$tsr = new TransactionSearchParameters();
$tsr->amountFrom="1";
$tsr->amountTo="20";
$tsr->cardLast4Digits="4507";

$transaction = new TransactionManager($merchant,$WsURL,$oauth_token);
$response = $transaction->findTransactions($tsr);
var_dump($response);
